index.bs

```diff
@@ -894,6 +912,65 @@ A {{Point2D}} represents a location in a two dimensional space. The origin of co
   </pre>
 </div>
 
+## Update camera focus distance and {{takePhoto()}} ## {#example4}
+
+<div class="example" highlight="javascript">
+  <pre>
+    &lt;html>
+    &lt;body>
+    &lt;video autoplay>&lt;/video>
+    &lt;img>
+    &lt;input type="range" hidden>
+    &lt;script>
+      var imageCapture;
+
+      navigator.mediaDevices.getUserMedia({video: true})
+        .then(gotMedia)
+        .catch(err => console.error('getUserMedia() failed: ', err));
+
+      function gotMedia(mediastream) {
+        const video = document.querySelector('video');
+        video.srcObject = mediastream;
+
+        const track = mediastream.getVideoTracks()[0];
+        imageCapture = new ImageCapture(track);
+
+        const capabilities = track.getCapabilities()
+        // Check whether focus distance is supported or not.
+        if (!capabilities.focusDistance) {
+          return;
+        }
+
+        // Map focus distance to a slider element.
+        const input = document.querySelector('input[type="range"]');
+        input.min = capabilities.focusDistance.min;
+        input.max = capabilities.focusDistance.max;
+        input.step = capabilities.focusDistance.step;
+        input.value = track.getSettings().focusDistance;
+
+        input.oninput = function(event) {
+          track.applyConstraints({
+            advanced : [{focusMode: "manual", focusDistance: event.target.value}]
```

Comment:
`advanced` is necessary bc of the way it's implemented in Chromium, and since for the examples sake the important thing is that they work, I'd say add a note here and please some one file a https://crbug.com/new ?